src/lightning_app/cli/lightning_cli.py

```diff
@@ -206,15 +290,63 @@ def stop():
     pass
 
 
-@_main.group(hidden=True)
+@_main.group(name="delete")
 def delete():
-    """Delete an application."""
+    """Delete a Lightning managed resources."""
     pass
 
 
-@_main.group(name="list", hidden=True)
+@delete.command("cluster")
+@click.argument("cluster", type=str)
+@click.option(
+    "--force",
+    "force",
+    type=bool,
+    required=False,
+    default=False,
+    is_flag=True,
+    help="""Delete a cluster from Lightning AI. This does NOT delete any resources created by the cluster,
+            it just removes the entry from Lightning AI.
+
+            WARNING: You should NOT use this under normal circumstances.""",
+)
+@click.option(
+    "--wait",
+    "wait",
+    type=bool,
+    required=False,
+    default=False,
+    is_flag=True,
+    help="Enabling this flag makes the CLI wait until the cluster is deleted.",
+)
+def delete_cluster(cluster: str, force: bool = False, wait: bool = False):
+    """Delete a Lightning AI compute cluster and all associated cloud provider resources.
+
+    Deleting a run also deletes all Runs and Experiments that were started on the cluster.
+    Deletion permanently removes not only the record of all runs on a cluster, but all associated experiments,
+    artifacts, metrics, logs, etc.
+
+    WARNING: This process may take a few minutes to complete, but once started it CANNOT be rolled back.
+    Deletion permanently removes not only the cluster from being managed by Lightning AI, but tears down
+    every resource Lightning AI managed (for that cluster id) in the host cloud.
+
+    All object stores, container registries, logs, compute nodes, volumes, etc. are deleted and cannot be recovered.
+    """
+    mgr = AWSClusterManager()
+    mgr.delete(cluster_id=cluster, force=force, wait=wait)
+
+
+@_main.group(name="list")
 def get_list():
-    """List your applications."""
+    """List your Lightning managed resources."""
+    pass
+
+
+@get_list.command("clusters")
+def list_clusters(**kwargs):
+    """List your Lightning AI compute clusters."""
```

Comment:
Please change to:

List your Lightning AI BYOC compute clusters.

Reply:
:+1: